### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Ghiftee**
In this milestone, I created forms to perform the following functions:
- Create a Post on behalf of the current_user.
- Create a comment on behalf of the current_user.
- Allow Users to add likes to Posts.

Please review my project 😁

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-# names = ['Juan', 'Pekka', 'Luula', 'Aarnio', 'Aarto']
+names = ['Juan', 'Pekka', 'Luula', 'Aarnio', 'Aarto']
 5.times do |i|
 	user = User.create(name: names[i], photo: "https://randomuser.me/api/portraits/men/jpg", bio: 'Lots of things to say in here')
```
Comment (oxenprogrammer) [inline db/seeds.rb:11]: **Optonal Changes**
- [x] The seed currently tries to submit a null value. Please let us try to fix it or simply remove it and add test data using `rails c`

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Ghiftee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] Please let us add the form to add post and also clicking on `like` button should be able to increase the count for like of a post.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ghiftee** [comment]
Thank you for the review @oxenprogrammer. I have implemented all the changes requested.
Let me give a brief summary of how the app works
On the home page, click on a username, you would be redirected to a page with details about the user where you would also see the create post button which creates a new post.
To comment or like a post, click on a post from the page with details about a user(bio, posts) and you would be redirected to a page where you can add a comment or like a post.
When you click on the like button, it will direct you back to the list of posts for the user and if you click on a post you already liked, it will show you a liked message

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Ghiftee , 

All forms work fine  :+1: 
I can like posts :+1: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ghiftee** [comment]
Thank you @sinansevgi

**Pull request merged**